src/SqlPersistence/SqlPersistence.csproj

```diff
@@ -10,7 +10,7 @@
 
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="[12.0.1, 13.0.0)" />
-    <PackageReference Include="NServiceBus" Version="[7.0.0, 8.0.0)" />
+    <PackageReference Include="NServiceBus" Version="[7.4.1, 8.0.0)" />
```

Comment:
Is this needed? It changes the minimum version of core for the package.

Reply:
Removed.